knotx-core/knotx-service-adapter/src/main/resources/knotx-service-adapter.json

```diff
@@ -0,0 +1,21 @@
+{
+  "config": {
```

Comment:
in the current approach, if you want to supply config to verticle while running verticle as standalon, you must not wrapp it to "config" -> otheriwse it will not work. See other verticles